.github/workflows/helm-vib.yaml

```
@@ -0,0 +1,46 @@
+name: Validate Helm Chart
+
```

```suggestion
  pull_request_target:
    branches:
      - main
```
Since we need to use the pull request target, according to @mpermar